playbooks/appserver.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Playbook to set up common services on any type of appserver.
+
+# Temporary: until we migrate all roles in all playbooks from `deploy-all.yml`, we install
+# Python here separately. Otherwise Ocim will complain about missing dependencies.
```

Comment:
It's actually coming before, which is the cause of the issue. I wanted to do it before to keep it consistent with the approach we took with common-server having consul and the node exporter in them. Do you think we should do it after or?

Reply:
We need to run it after edx_sandbox – otherwise neither Python 2 nor nginx will be available.  You found a fix for the lack of Python 2, but there is no easy fix for the missing nginx.

Reply:
Ah, right. Damn :) I'll make that change soon. As a side note for the future -- once we consolidate roles, should we try moving that `nginx.yml` in common-server into its own role?